astro/src/diagrams/docs/lifecycle/authenticate-users/tenant-mfa-decision-tree.astro

```diff
@@ -0,0 +1,34 @@
+---
+import Diagram from 'src/components/mermaid/FlowchartDiagram.astro';
+
+const { alt } = Astro.props;
+
+//language=Mermaid
+const code = `
+graph TD
+    applicationMfaPolicy[MFA Policy At Application Level Configured?] --> |True| applicationMfaPolicyControls[Application MFA Policy Controls]
+    applicationMfaPolicy --> |False| tenantMfaPolicy
+    tenantMfaPolicy[What is MFA Policy At Tenant Level?] --> |Disabled| noChallenge[Do Not Challenge User]
+    tenantMfaPolicy --> |Enabled| checkUserConfigEnabled[Check User Has MFA Method]
+    tenantMfaPolicy --> |Required| checkUserConfigRequired[Check User Has MFA Method]
+    checkUserConfigEnabled --> |MFA Configured| contextualMFACheck[Check Request Context]
```

Comment:
The children of the two `MFA Configured` nodes are identical. Is it possible for `Enabled` and `Required` branches to converge instead of repeating or would that make the diagram more difficult to read?
<img width="965" alt="image" src="https://github.com/user-attachments/assets/020e1d39-cadb-4427-a692-0295c23dbd88" />